Dockerfile_k8s_dev

```diff
@@ -0,0 +1,18 @@
+FROM registry.suse.com/bci/python:3.10
+
+RUN zypper -n in python310-devel gcc tar gzip kubernetes1.23-client
+
+# Goole cli installation
+RUN cd /opt && curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-415.0.0-linux-x86_64.tar.gz && tar -xf google-cloud-cli-415.0.0-linux-x86_64.tar.gz && rm -f google-cloud-cli-415.0.0-linux-x86_64.tar.gz && google-cloud-sdk/bin/gcloud components install gke-gcloud-auth-plugin
+
+# Install python dependences
+COPY requirements_k8s.txt requirements_k8s_test.txt /pcw/
+RUN  pip3.10 install -r /pcw/requirements_k8s_test.txt && rm -rf /var/cache
+
+# Copy program files only
+COPY ocw  /pcw/ocw/
```

Comment:
I assume this COPY should be removed from devel container so you will bind sources via volume